src/countBulls.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+function countBulls(inputNumber, gameNumber) {
+  const inputNumberItems = inputNumber
+    .toString().split('').map(Number);
+  const gameNumberItems = gameNumber
+    .toString().split('').map(Number);
+  let bulls = 0;
+
+  gameNumberItems.forEach((item, index) => {
+    if (item === inputNumberItems[index]) {
+      bulls++;
+    }
+  });
+
+  return bulls;
+}
```

Comment:
that would work the same with valid input
```suggestion
function countBulls(inputNumber, gameNumber) {
  const inputNumberItems = inputNumber
    .toString();
  const gameNumberItems = gameNumber
    .toString().split('');
  let bulls = 0;

  gameNumberItems.forEach((item, index) => {
    if (item === inputNumberItems[index]) {
      bulls++;
    }
  });

  return bulls;
}
```